frontend/src/features/properties/list/columns.tsx

```diff
@@ -69,7 +68,7 @@ export const columns = (
     sortable: true,
     filterable: true,
     filter: {
-      component: ParentGroupedFilter,
+      component: TypeaheadField,
```

Comment:
Outdated parent grouped filter, as well as we do not need it as we are splitting up the agency and sub agencies here